addons/loyalty_delivery/views/loyalty_reward_views.xml

```diff
@@ -5,11 +5,29 @@
         <field name="model">loyalty.reward</field>
         <field name="inherit_id" ref="loyalty.loyalty_reward_view_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//group[@id='product_group']" position="after">
-                <group id="shipping" attrs="{'invisible': [('reward_type', '!=', 'shipping')]}">
+            <xpath expr="//group[@id='reward_type_group']" position="after">
+                <group string="Free shipping" attrs="{'invisible': [('reward_type', '!=', 'shipping')]}">
```

Comment:
Don't remove the id it could always be useful
```suggestion
                <group id="shipping" string="Free shipping" attrs="{'invisible': [('reward_type', '!=', 'shipping')]}">
```